src/vsr/replica.zig

```diff
@@ -3970,36 +3659,43 @@ pub fn ReplicaType(
             }
 
             // Request any missing or disconnected headers:
-            // TODO Snapshots: Ensure that self.commit_min op always exists in the journal.
-            var broken = self.journal.find_latest_headers_break_between(self.commit_min, self.op);
-            if (broken) |range| {
-                log.debug("{}: repair: break: view={} op_min={} op_max={} (commit={}..{} op={})", .{
-                    self.replica,
-                    self.view,
-                    range.op_min,
-                    range.op_max,
-                    self.commit_min,
-                    self.commit_max,
+            if (self.commit_min != self.op) {
+                var broken = self.journal.find_latest_headers_break_between(
```

Comment:
`commit_min` usually exists, but _not always_. And if it doesn't exist, we don't actually want to repair it: the case were it doesn't exist is when it is `op_checkpoint` and has been overwritten (or may have been overwritten — if there is corruption we might not be able to tell which).